CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 1.1.3~dev (Unreleased)
 
+* Fixed a crash that occurs when trying to add/edit/test LaTeX if LaTeX is not found (#4334, #4345)
+* Fixed touch enabling on quit, preventing a crash in some cases (#4343, #4373)
+* Fixed a bug that closed text fields immediately after creating them with the middle mouse button (#3983, #4309)
```

Comment:
This fix affects more than just middle mouse:
```suggestion
* Fixed a bug that closed text fields immediately after creating them with a secondary mouse or stylus button (#3983, #4309)
```

Reply:
I'm fine with the suggestion.